Frontend/ui-customer/src/Screens/CustomerUtils/FavouriteDriverTrips/View.purs

```diff
@@ -0,0 +1,263 @@
+import Language.Strings (getString)
+import Language.Types (STR(..))
+import Mobility.Prelude (capitalize)
+import Prelude ((<>), show)
+import Prelude (Unit, const, map, unit, ($), (&&), (/=), (<<<), (<=), (<>), (==), (/), not, (-), (||))
+import PrestoDOM (Accessiblity(..), FlexWrap(..), Gravity(..), Length(..), Margin(..), Orientation(..), Padding(..), PrestoDOM, Screen, Visibility(..), accessibility, accessibilityHint, adjustViewWithKeyboard, afterRender, alignParentBottom, background, color, cornerRadius, disableClickFeedback, editText, fontStyle, frameLayout, gravity, height, hint, hintColor, imageUrl, imageView, imageWithFallback, layoutGravity, linearLayout, margin, multiLineEditText, onBackPressed, onChange, onClick, orientation, padding, pattern, relativeLayout, scrollView, stroke, text, textSize, textView, visibility, weight, width, onAnimationEnd, alpha, textFromHtml)
+import Screens.CustomerUtils.FavouriteDriverTrips.Controller (Action(..), ScreenOutput, eval)
+import Screens.Types as ST
+import Styles.Colors as Color
+import Effect.Aff (launchAff)
+import Services.API (FavouriteDriverTripsResp(..))
+import Control.Monad.Except (runExceptT)
+import Control.Monad.Trans.Class (lift)
+import Control.Transformers.Back.Trans (runBackT)
+import Engineering.Helpers.Utils as EHU
+import Services.FlowCache as FlowCache
+import Types.App (GlobalState(..), defaultGlobalState, FlowBT)
+import Helpers.Pooling (delay)
+import Effect.Aff (Milliseconds(..))
+import Services.Backend as Remote
+import Data.Either
+import JBridge as JB
+import Data.Array (range)
+import Mobility.Prelude (boolToVisibility)
+
+screen :: ST.FavouriteDriverTripsState -> GlobalState -> Screen Action ST.FavouriteDriverTripsState ScreenOutput
+screen initialState st =
+  { initialState
+  , view
+  , name : "FavouriteDriverTrips"
+  , globalEvents : [
+    (\push -> do
+        _ <- launchAff $ EHC.flowRunner defaultGlobalState $ runExceptT $ runBackT $ getFavouriteDriverTrips push initialState
+        pure $ pure unit
+      )
+    ]
+  , eval : \state  action -> do
+      let _ = spy  "FavouriteDriverTrips action " state
+      let _ = spy  "FavouriteDriverTrips state " action
+      eval state action
+  }
+
+getFavouriteDriverTrips :: (Action -> Effect Unit) -> ST.FavouriteDriverTripsState -> FlowBT String Unit
+getFavouriteDriverTrips push state = do
+  void $ lift $ lift $ EHU.toggleLoader true
+  (favouriteDriversResp) <- lift $ lift $ Remote.getFavouriteDriverTrips "5" "0" "false" state.data.driverNumber
+  case favouriteDriversResp of
+    Right resp -> do
+      void $ lift $ lift $ delay $ Milliseconds 2000.0
+      void $ lift $ lift $ EHU.toggleLoader false
+      liftFlowBT $ push $ GetFavouriteDriversTripsAPIResponseAction (resp)
+      pure unit
+    Left _ -> do
+      void $ pure $ JB.toast $ getString NOT_AVAILABLE
+      liftFlowBT $ push $ BackPressed
```

Comment:
If api failed we need to have a proper error message instead of BackPressed

Reply:
yea, we are displaying toast Not available at this moment in case of failure